portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile/MyTestFile.testcase

```diff
@@ -0,0 +1,31 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+property testray.main.component.name = "Forms";
+setUp {
+TestCase.setUpPortalInstance();
+User.firstLoginPG();
+Navigator.openURL();
+}
+tearDown {
+User.logoutAndLoginPG(
+userLoginEmailAddress = "test@liferay.com",
+userLoginFullName = "Test Test");
+}
+
+@description = "Access Portal Global Menu and validate CONFIGURATION string at
+Control Panel"
+@priority = "5"
+test ValidateConfigurationStringAtControlPanel {
+task("will open the application menu"){
+        Click(locator1 = "//button[contains(@data-qa-id, 'applicationsMenu')]");
+}
+task("access the control panel option"){
+        Click(locator1 = "//*[text()='Control Panel']");
+}
+task("checking if the configuration option is in the control panel section"){
+        AssertVisible(locator1 = "//h2[text()='Configuration']");
+}
+
+
+}
+}
```

Comment:
This commit(https://github.com/lucasperj/liferay-portal/commit/068a835d3ed696c0e79a561b861ada02390c8687) does not apply to this ticket, can you please remove and resend? Thanks.